perfmetrics/scripts/continuous_test/ml_tests/tf/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
do you have a document specifying the kokoro setup - what files are there, how are we running etc? add that document in readme. 
If you dont have any document specifying the setup, add all the information in readme file.